docs/sources/mimir/configure/configure-ingester-circuit-breakers.md

```diff
@@ -0,0 +1,89 @@
+---
+aliases:
+  - ../configuring/configuring-ingester-circuit-breakers/
+description: Learn how to configure ingester circuit breakers.
+menuTitle: Configure ingester circuit breakers
+title: Configure Grafana Mimir ingester circuit breakers
+---
+
+# Configure Grafana Mimir ingester circuit breakers
+
+## Background
+
+Use circuit breakers to prevent an application from repeatedly trying to run an operation that is likely to fail.
+A circuit breaker monitors the number of recent failures and then uses this information to decide whether to allow a new operation to proceed, or to simply return an exception immediately.
+In the case of a failing operation, a circuit breaker allows an application to proceed, without waiting for you to resolve the failure cause.
+Because the failing operation is immediately rejected, the application doesn't retry it. This reduces the application's CPU usage.
```

Comment:
can we say "Mimir" instead of "application?" Maybe even "the ingester" instead of "Mimir." It wasn't clear on a first read if an application is Mimir's client or Mimir itself.

I'd also mention somewhere (maybe not in this paragraph) that even if the operation fails in the ingester, the overall operation might still success because of the replication of data on ingesters. This is obvious to us, but may not be to someone new to Mimir.

Reply:
This is a general paragraph in which we explained how circuit breakers work. It is not related to Mimir only. That's why we opted for "application".